src/demokratie.md

```diff
@@ -292,30 +292,30 @@ Geschichte der Demokratie bisher keine große Rolle gespielt zu haben,
 
 ### Sind Blockaden und Reformstau etwas Schlechtes?
 
-Je nachdem, wie man ein politisches System gestaltet, kann es, solange man
-Mechanismen hat, die das Diktat der Mehrheit verhindern, zu theoretisch
-unendlich-langen Blockaden kommen, welche "Reformstau" könnten. Bei genauerer
+Je nachdem wie man ein politisches System gestaltet kann es, dass solange man
+Mechanismen hat, die das Diktat der Mehrheit verhindern, es zu theoretisch
+unendlich-langen Blockaden, einem "Reformstau" kommen könnte. Bei genauerer
```

Comment:
Hier stimmt etwas nicht, zumindest ergibt der neue Satz fuer mich keinen Sinn mehr.
Allerdings ist der letzte Nebensatz bei in meiner Version auch nicht korrekt, daher hier ein Vorschlag (ich habe versucht, diesen langen Satz zu zerlegen. Damit sollte es ein bisschen verstaendlicher sein):

```suggestion
Es kann es, solange man Mechanismen hat, die das Diktat der Mehrheit verhindern, theoretisch
zu unendlich-langen Blockaden kommen. Das Resultat ist dann: "Reformstau". Bei genauerer
```

Die Kommata scheinen fuer mich korrekt zu sein.

Reply:
Das hier fehlt noch